src/blocks/collapsible/collapsible.js

```diff
@@ -106,6 +115,8 @@ registerBlockType( 'bu/collapsible', {
 			customMarginBottom,
```

Comment:
@acketon After thinking about it, "customSpacing" might be better name for this toggle? what do you think 
cc @ashleykolodziej

Reply:
Ah, sorry @acketon you already merged it.
All good from my end.
Sorry for the late CR.